Server/routes/follows.js

```diff
@@ -0,0 +1,104 @@
+/*
+Server Follows Route | SUITAPP Web App
+GROUP 1: Amine Bensalem, Douglas MacKrell, Savita Madray, Joseph P. Pasaoa
+*/
+
+
+/* MODULE INITS */
+const express = require('express');
+const router = express.Router();
+// Queries
+const { 
+  getFollows,
+  getFollowers,
+  createFollow,
+  deleteFollow
+} = require('../Models/follows.js');
+
+
+/* HELPERS */
+const handleError = (req, res, error) => {
+  console.log(error);
+  res.status(500);
+  res.json({
+      status: "fail",
+      message: "error: backend issue",
+      payload: null
+  });
+}
+
+const checkNumId = (req, res, whichId) => {
+  const { [whichId] } = req.params;
+  if (![whichId] || isNaN(parseInt([whichId].trim()))) {
+    return handleError(req, res, `invalid ${whichId} parameter`);
```

Comment:
I think it's more appropriated to do an if-else
if .. you call the error function and return false
else ... return true

the rest will be explained in my next comment

Reply:
hmm, okay, ill give this a good look over when i can be in front of my laptop for long